lib/vmfloaty.rb

```diff
@@ -184,10 +184,12 @@ def run # rubocop:disable Metrics/AbcSize
     end
 
     command :delete do |c|
-      c.syntax = 'floaty delete hostname,hostname2 [options]'
+      c.syntax = 'floaty delete hostname,hostname2 [options]
+    floaty delete job1,job2 [options] (only supported with ABS)'
```

Comment:
^^indentation?

Reply:
Futzed around with it a bit and found a better way of adding to the syntax that can be aligned correctly.